Source/WebCore/dom/NodeInlines.h

```diff
@@ -40,10 +40,11 @@ inline RefPtr<ScriptExecutionContext> Node::protectedScriptExecutionContext() co
 
 inline WebCoreOpaqueRoot Node::opaqueRoot() const
 {
+    if (isConnected()) {
+        Locker locker { TreeScope::treeScopeMutationLock() };
+        return WebCoreOpaqueRoot { &treeScope().documentScope() };
+    }
     // FIXME: Possible race?
```

Comment:
Does it make this comment stall ?

Reply:
No, that comment still applies to `traverseToOpaqueRoot()`.